### PR TITLE
Launch date

### PR DESCRIPTION
**Pull request by Chilipp**
This PR adds the two kickoff events to the website. You can preview them here, @ClaireWyatt please review them carefully and check for any mistakes (particularly with respect to affiliation, etc.):

https://911-267395254-gh.circle-artifacts.com/0/SORSE/programme/talks/event-001/index.html
https://911-267395254-gh.circle-artifacts.com/0/SORSE/programme/talks/event-002/index.html

They are also shown in the list of [upcoming events][upcoming] and in the [calendar][calendar]

I also made a new post for the kickoff based on the text by @ClaireWyatt: https://911-267395254-gh.circle-artifacts.com/0/SORSE/programme/kickoff/index.html

There are still some todos. We need

- ~~the abstract by Kari Jordan~~ will be postponed to a later PR
- [x] to show the introductory talk in the calendar
- [x] to adjust the order and exact timings of the talks

@ClaireWyatt, for the introductory talk, shall I make a separate event page? Or shall we just add a 15-minutes(?) slot to the calendar? I'd prefer to have an event page, but this needs a little abstract and info about the speaker.

@eileen-kuehn and @cc-a: How will the registration take place? Will one of you create an event in indico? I think it should be only one event, not two, right?

@eileen-kuehn: These events have not been submitted via indico as they are invited talks. Do you see any problems that I name them `event-001` and `event-002`? 

[upcoming]: https://911-267395254-gh.circle-artifacts.com/0/SORSE/index.html#upcoming-events
[calendar]: https://911-267395254-gh.circle-artifacts.com/0/SORSE/programme/calendar/index.html

### CODE REVIEW

**Comment by eileen-kuehn** [comment]
@Chilipp, @ClaireWyatt, the event lasts from 13:00-15:00, we do have
* Welcoming words (10 mins?)
* Presentation 1 (30 mins)
* Presentation 2 (30 mins)

How do we make the timings? Does the first presentation start at 13:10? What is scheduled after the presentations? Does a discussion take place?

**Comment by cc-a** [comment]
A [registration link](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.google.com%2Fforms%2Fd%2Fe%2F1FAIpQLSetlGKtfCxxaAjtHEGxaQ58o360tn9y5BTqHypvc2qnly5CnQ%2Fviewform&data=01%7C01%7Cc.wyatt%40soton.ac.uk%7C7665ed066c8a4978a60908d8446852c3%7C4a5378f929f44d3ebe89669d03ada9d8%7C0&sdata=2qD1tm8gydJF41gajsozyg2FdCg3%2BdFfz3JbhNki6Nw%3D&reserved=0) has already gone out with the announcement. We weren't able to to set up an Indico event in time.

**Comment by Chilipp** [comment]
hmm ok @cc-a. this is not [what we say on the website](https://sorse.github.io/faq/about/costs)... 

But now we have to go for this. My apologies @eileen-kuehn, I did not realize that a registration link was already in the announcement email.

**Comment by cc-a** [comment]
Apologies, we did want to go with Indico but with Claire going on annual leave from today things  were a bit rushed and nobody with access to Indico was available. We should be sticking with Indico for event registration going forward.

As Claire is unavailable would you like me to review this?

TBH I think what we use to do registration is a bit of an implementation detail and probably doesn't need to be mentioned on the website anyway...

**Comment by Chilipp** [comment]
> As Claire is unavailable would you like me to review this?

yes please @cc-a, thanks :smiley:  

> TBH I think what we use to do registration is a bit of an implementation detail and probably doesn't need to be mentioned on the website anyway...

well, it's important in terms of the [privacy notice](https://sorse.github.io/privacy/#event-registration) because we have to declare what services we use and what happens with the sensible information, such as email-adresses, etc.

File: _events/talks/event-001.md
Comment (cc-a) [inline _events/talks/event-001.md:24]:
```suggestion
Coming soon...
```

**Review by cc-a — APPROVED** [review]
The really key thing that should go up ASAP is the kickoff page. If we think the individual event pages aren't ready to go yet (due to missing abstract + possible changes to timing) then suggest these are deferred for now. Personally I think these can go up now and if we tweak things to add an intro event later this is unlikely to cause any problems.

If we're going ahead with publishing the individual events now please also add one for the networking session afterwards.

The date information in the  "Add to Calendar" link for Office 365 doesn't seem to work correctly.

**Comment by Chilipp** [comment]
sounds good to me, thanks @cc-a! Do you know anything about the lengths of these sessions? Is the approach by @eileen-kuehn from [above](https://github.com/SORSE/sorse.github.io/pull/282#issuecomment-677123971) valid?

> - Welcoming words (10 mins)
> - Presentation 1 (30+10 mins)
> - Presentation 2 (30+10 mins)
> - Discussions & Networking (30 mins)

**Comment by Chilipp** [comment]
> The date information in the "Add to Calendar" link for Office 365 doesn't seem to work correctly.

@cc-a, unfortunately I do not have an account there to debug it. Could you be a bit more explicit?

**Comment by cc-a** [comment]
Timings look about right.

Calendar link tried to create an event but date/time info is missing and it defaults to the current time. Perhaps a separate issue can be created for this to avoid blocking this pr.

**Comment by Chilipp** [comment]
> Perhaps a separate issue can be created for this to avoid blocking this pr.

Agreed, thanks. Can you open it? Ideally with a screenshot. I'll add the other events to the calendar.

**Comment by Chilipp** [comment]
@cc-a, I added the complementary events to the website and updated the links in [my above comment](https://github.com/SORSE/sorse.github.io/pull/282#issue-470462495). From my perspective, this is then ready to be merged and we'll make a new PR when we have the abstract by Kari Jordan. Please approve this PR if you agree to this.

I wonder whether we should highlight in the google form this is an exception and therefore deviates from our privacy notice.

**Review by cc-a — APPROVED** [review]
Looks good! Thanks all.

**Pull request merged**
